Install SELinux policy to active one

engelmi opened the pull request:
Relates to: https://github.com/containers/qm/pull/731

Change the bluechi-selinux module to be installed to the targeted policy (which is hardcoded), simply install it to the active SELinux policy.

engelmi committed:
Install SELinux policy to active one

Relates to: https://github.com/containers/qm/pull/731

Change the bluechi-selinux module to be installed to the targeted
policy (which is hardcoded), simply install it to the active SELinux policy.

Signed-off-by: Michael Engel <mengel@redhat.com>

mwperina reviewed (approved):
LGTM

alexlarsson commented on bluechi.spec.in:
```diff
@@ -186,14 +184,14 @@ SELinux policy associated with the bluechi and bluechi-agent daemons
    semodule -N -X 200 -r hirte 2>/dev/null || true
 fi
-%selinux_modules_install -s %{selinuxtype} %{_datadir}/selinux/packages/bluechi.pp.bz2
+%selinux_modules_install -s ${SELINUXTYPE} %{_datadir}/selinux/packages/bluechi.pp.bz2
```
This is not enough, you need to source /etc/selinux/config in order to know what $SELINUXTYPE is.

alexlarsson replied:
I also think you need to Requires(post) the policy package so the postscripts are run in the correct order.
This is what I did in qm:
```
Requires(post): selinux-policy-base >= %_selinux_policy_version
Requires(post): selinux-policy-any >= %_selinux_policy_version

```

engelmi replied:
Updated :heavy_check_mark: PTAL @alexlarsson

engelmi committed:
Make haproxy in selinux policy optional

The haproxy types are not (yet) available in the automotive selinux policy.
In order not to break the bluechi policy, the haproxy part has been wrapped
in an optional policy block.

Signed-off-by: Michael Engel <mengel@redhat.com>
